contact.html

```diff
@@ -0,0 +1,119 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&family=Quicksand:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="css/main.css" />
+
+    <title>Responsive grid project</title>
+  </head>
+
+  <body>
+    <!-- Add your markup here -->
+    <header>
+        <div class="main_container">
+          <div class="square_icon" id="square_2">
+            <div class="square_1">
+              <div id="square_2inlineContent" data-testid="inline-content" class="">
+                <div
+                  id="square_2inlineContent-gridContainer"
+                  data-testid="mesh-container-content"
+                ></div>
+              </div>
+            </div>
+            </div>
+            <div class="name_container">
+              <span class="font1 name bigger"> Maya Nelson </span>
+              <span class="font2 title"> PROJECT MANAGER </span>
+            </div>
+            <div class="icon_container_1 menu_icon">
+              <div class="icon_line_1 icon_line_1_2 icon_line_1_1"></div>
+              <div class="icon_line_1 _2vsXM"></div>
+              <div class="icon_line_1 icon_line_1_2 icon_line_1_3"></div>
+            </div>
+            <div class="nav_container" >
+                <nav>
+                  <ul class="font2" style="font-size: 13px;">
+                    <li>
+                      <a href="/index.html">ABOUT ME</a>
+                      </li>
+                        <li>
+                          <a href="#">RESUME</a>
+                        </li>
+                        <li>
+                          <a href="#">PROJECTS</a>
+                        </li>
+                        <li>
+                          <a href="/contact.html">CONTACT</a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+    </div>
+    </header>
+
+    <main id="contact_container" style="background-image:var(--grey1);background-color: var(--grey1);border-bottom: dotted; border-bottom-color: transparent;">
+        <section class="contact_header">
+            <div class="square_icon" id="square_2">
+                <div id="align" class="square_1" style="margin: inherit;">
+                  <div id="square_2inlineContent" data-testid="inline-content" class="">
+                    <div
+                      id="square_2inlineContent-gridContainer"
+                      data-testid="mesh-container-content"
+                    ></div>
+                  </div>
+                </div>
+              </div>
+
+            <div id="bigger" class="contact_slogan font1">Let's talk</div>
```

Comment:
instead of using both an id and class and using each within separate media queries, try to just use one, for example your class and target it within both media queries :)